HHS/OPRE-OPS
# feat: role-based filtering of My CANs

## What changed

- adds role-based filtering for My CANs tab

## Issue

#329 

## How to test

1. login as admin
2. add basic user to some agreements
3. logout
4. login as basic user
5. goto my CANs
6. see cans based on agreement team membership

## Screenshots

https://github.com/user-attachments/assets/ea76111d-3533-4ead-b5e5-dec52d2c8053

## Definition of Done Checklist
